test_cases/test_dojo.py

```diff
@@ -43,4 +45,34 @@ def test_create_room_returns_error_message_if_roomname_not_string(self):
 		with self.assertRaises(ValueError):
 			self.the_dojo.create_room("OFFICE",123)
 	
+	def test_that_a_room_exists_before_printing_occupants(self):
+		self.the_dojo.create_room("office","Green")
+		self.assertTrue(self.the_dojo.print_room("Green"))
```

Comment:
Great idea to test this. However, the print function returns `None` unless you specifically command it to return a value. To carry out a unit test on you must "capture"  the output and then compare it with the expected output (in this case "Green")